### DISCUSSION TITLE
gongju/warp

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#187] [Announcements]
# gongju/warp

Warp：一款21世纪人用的终端工具，GitHub星标2.8k+，用完爱不释手

https://tobebetterjavaer.com/gongju/warp.html

**Comment by zzzpppy** [comment]
我的ls 就报错

**Comment by zuilintan** [comment]
请仔细看下它的仓库, 这叫开源吗?
